src/nnet3/nnet-test-utils.cc

```diff
@@ -984,6 +984,12 @@ static void GenerateRandomComponentConfig(std::string *component_type,
       os << "new-column-order=" << buffer.str();
       break;
     }
+    case 21: {
+      *component_type = "PerElementOffsetComponent";
+      os << "dim=" << RandInt(1, 100)
+         << " learning-rate=" << learning_rate << " param-mean=0.0 param-stddev=1.0";
```

Comment:
So I should also change the initialization that I have added here to "param-mean=0.0 param-stddev=0.0" ? 
Wouldn't that initialize offsets to all zero ? Is that ok for the test ?